mcs/class/System.Data.OracleClient/Test/testblob.cs

```diff
@@ -9,7 +9,7 @@ class TestBlob
 {
 	static string infilename = @"../../../tools/mono-win32-setup-dark.bmp";
 	static string outfilename = @"mono-win32-setup-dark2.bmp";
-	static string connectionString = "Data Source=testdb;User ID=scott;Password=tiger";
+	static string connectionString = "Data Source=testdb;User ID=scott;Password=PLACEHOLDER";
```

Comment:
other areas in code base also have same password : https://github.com/mono/mono/search?q=Password%3Dtiger - might be great to update all at same time.